idd/Energy+.idd.in

```diff
@@ -32499,6 +32795,170 @@ DesignSpecification:OutdoorAir,
       \note This input is only used to calculate the minimum outdoor air flow rate when the field
       \note System Outdoor Air Method = ProportionalControlBasedOnDesignOARate in Controller:MechanicalVentilation.
 
+DesignSpecification:OutdoorAir:SpaceList,
```

Comment:
I don't see this object in the example file, used elsewhere in a different example file?

Reply:
Regarding the Space-1B test, that's a little complicated and I may have to change the rules. The way it is currently if any surface in Zone 1 has a space specified, any other surfaces in the zone with a blank space are assigned to an auto-generated space named "Zone 1-Remainder". If no surfaces in Zone 1 are assigned to a space, then the auto-generated space is named "Zone 1".

The current new example file isn't the latest. Here's a better one that will be used instead (next push). This one also uses DesignSpecification:OutdoorAir:SpaceList. 
[5ZoneAirCooledWithSpaces-Jan.idf.txt](https://github.com/NREL/EnergyPlus/files/7015377/5ZoneAirCooledWithSpaces-Jan.idf.txt)